code/modules/roguetown/roguejobs/artificer/handsaw_chisel.dm

```diff
@@ -0,0 +1,179 @@
+#define BCLASS_CHISEL			"chisel"
+//................	Handsaw	............... //
+/obj/item/rogueweapon/handsaw
+	force = 5
+	possible_item_intents = list(/datum/intent/mace/strike/shovel)
+	gripped_intents = null
+	name = "handsaw"
+	desc = "Iron tool for woodworking."
+	icon_state = "handsaw"
+	item_state = "handsaw"
+	icon = 'icons/roguetown/items/crafting.dmi'
+	mob_overlay_icon = 'icons/roguetown/onmob/onmob.dmi'
+	lefthand_file = 'icons/roguetown/onmob/lefthand.dmi'
+	righthand_file = 'icons/roguetown/onmob/righthand.dmi'
+	experimental_inhand = FALSE
+	experimental_onhip = FALSE
+	grid_width = 32
+	grid_height = 96
+	sharpness = IS_SHARP
+	wdefense = 0
+	wlength = WLENGTH_LONG
+	w_class = WEIGHT_CLASS_NORMAL
+	slot_flags = ITEM_SLOT_HIP
+	swingsound = list('sound/combat/wooshes/blunt/shovel_swing.ogg','sound/combat/wooshes/blunt/shovel_swing2.ogg')
+	drop_sound = 'sound/foley/dropsound/shovel_drop.ogg'
+	smeltresult = /obj/item/ingot/iron
+	associated_skill = /datum/skill/combat/maces
+	max_blade_int = 50
+
+//................	Chisel	............... //
+/obj/item/rogueweapon/chisel
+	name = "chisel"
+	desc = "Add something to strike it with before doing stonework. Like a mallet or a stone."
+	icon_state = "chisel"
+	icon = 'icons/roguetown/items/crafting.dmi'
+	lefthand_file = 'icons/roguetown/onmob/lefthand.dmi'
+	righthand_file = 'icons/roguetown/onmob/righthand.dmi'
+	experimental_inhand = FALSE
+	experimental_onhip = FALSE
+	grid_width = 32
+	grid_height = 64
+	force = 10
+	throwforce = 2
+	possible_item_intents = list(/datum/intent/stab)
+	sharpness = IS_SHARP
+	dropshrink = 0.9
+	w_class = WEIGHT_CLASS_SMALL
+	wdefense = 0
+	blade_dulling = 0
+	max_integrity = 140
+	slot_flags = ITEM_SLOT_HIP
+	swingsound = list('sound/combat/wooshes/blunt/shovel_swing.ogg','sound/combat/wooshes/blunt/shovel_swing2.ogg')
+	drop_sound = 'sound/foley/dropsound/shovel_drop.ogg'
+	associated_skill = /datum/skill/combat/maces
+	max_blade_int = 50
+	dropshrink = 0.9
+
+/obj/item/rogueweapon/chisel/attackby(obj/item/W, mob/living/user, params)
+	. = ..()
+	if(istype(W,/obj/item/natural/stoneblock))
```

Comment:
actually coping and seething over this

Reply:
I regret nothing

Reply:
this pr should probably be frozen until slapcrafting is either closed or added since it does this but in a modular way

Reply:
To be fair made this for V originally but since the sleeve PR never got merged I lost patience and stopped adapting it for slaprecipes. It will happen at some point. Whether this repo finishes the slap port or not is another matter, I got no clue.